testing/python/fixture.py

```diff
@@ -336,6 +336,38 @@ def test_spam(spam):
         result = testdir.runpytest(testfile)
         result.stdout.fnmatch_lines(["*3 passed*"])
 
+    def test_override_autouse_fixture_with_parametrized_fixture_conftest_conftest(self, testdir):
```

Comment:
Yes, it was deliberate. I've just followed the convention in the file from other tests.